pkgs/tools/networking/openssh/default.nix

```diff
@@ -46,6 +46,7 @@ stdenv.mkDerivation rec {
   # I set --disable-strip because later we strip anyway. And it fails to strip
   # properly when cross building.
   configureFlags = [
+    "--sbindir=\${out}/bin"
```

Comment:
I think this is unnecessary. The default fixup moves `$out/sbin` to `$out/bin` or does that not happen here for some reason?